pkg/stellar/stellar.go

```diff
@@ -131,11 +133,23 @@ func (w *Wallet) CreateAccount() (string, string, error) {
 		return "", "", errors.Wrap(err, "failed to get source account")
 	}
 
-	err = w.activateEscrowAccount(newKp, sourceAccount, client)
-	if err != nil {
-		return "", "", errors.Wrapf(err, "failed to activate escrow account %s", newKp.Address())
+	activateEscrowBackoff := func() error {
+		log.Info().Msg("Trying to activate escrow account")
+		return w.activateEscrowAccount(newKp, sourceAccount, client)
 	}
 
+	bo := backoff.NewExponentialBackOff()
+	bo.MaxElapsedTime = time.Minute * 5 // retry for 5 minutes
```

Comment:
I still think this is way to much. Imagine waiting 5 minute on an REST call. HTTP proxy will even usually cur the connection before this.
I would go for something like 5 to 10 second  max total and a interval between retry of 1 sec max

Reply:
True, but the activation call already takes over 1 second to submit to horizon. I would retry every 2 seconds then for a total of 20 sec?

Reply:
I don't think the amount of time during the function execution is taken in account during the callback. The 1 second internal is the amount of time the program sleep before retrying.

Reply:
okay :+1: